runtime/stdlib/flow.go

```diff
@@ -209,6 +209,21 @@ var AccountEventContractParameter = sema.Parameter{
 	TypeAnnotation: sema.StringTypeAnnotation,
 }
 
+var AccountEventHashAlgorithmParameter = sema.Parameter{
+	Identifier:     "hashAlgorithm",
+	TypeAnnotation: sema.UInt8TypeAnnotation,
```

Comment:
not sure if this is the correctly to identify a enum

Reply:
Enums are essentially structs with an `rawValue` field. It's probably better to emit an event with more type information, i.e.

```suggestion
	TypeAnnotation: sema.HashAlgorithmTypeAnnotation,
```

The code in `newAccountKeysAddFunction` and in the test already is correct, only the type information here is wrong.